README.md

```diff
@@ -1,16 +1,106 @@
-new [cisagov](https://github.com/cisagov) GitHub project started.
+new [cisagov](https://github.com/cisagov) Go project started.
 This skeleton project contains [licensing information](LICENSE), as
 well as [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for the major languages that we use.
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+## Installation ##
+
+To install the example package, use `go build`:
+
+```console
+cd ./skeleton-golang-package/src/example
+go build
+```
+
+This will produce an executable `example` within the current directory.
+
+Execute the `./example` binary to run the program.
+
+You should see the contents of the main package as defined in [main.go](src/example/main.go).
+
+### Nix ###
+
+If you have [Nix](https://nixos.org/download.html) installed you can use
+the [flake.nix](flake.nix) configuration file located at the root of the
+project to build and run the `example` package:
+
+```console
+cd ./skeleton-golang-package
+nix run
+```
+
+After the build has completed the program will automatically execute. The build
+will also have created a `result` directory within the root of the project.
+This directory contains a `bin` subdirectory where the executable `example` can
+be found: `result/bin/example`
+
+To run the program simply execute the binary from the root of the `skeleton-golang-package`:
+
+```console
+./result/bin/example
+```
+
+Alternatively, if you want to execute an interactive build environment
+with all available dependencies for this project you can run this
+command in the project root:
+
+```console
+cd ./skeleton-golang-package
+nix develop
+```
+
+If you make changes to the project files you will need to update `vendorSha256`
+in `flake.nix`  You can do this by running `nix build` and you should be presented
+with an error similar to:
+
+```console
+error: hash mismatch in fixed-output derivation '/nix/store/11a71sp1wynjgxinx6yb0yhli4q659zi-example-20230424-go-modules.drv':
+         specified: sha256-+aBatR0sHWkykrQr8AMD0P5xoc4VEXtA+egp6PxsKzY=
+            got:    sha256-gqfvjULp2VApWQl7yFVj45meYpcS4XefUtEUy+TtAH4=
+error: 1 dependencies of derivation '/nix/store/nm6xhgfawf67sy89rl9azdzvnjc5r7cr-example-20230424.drv' failed to build
+```
+
+You would then copy the `got` hash and use it to replace the previous
+ `vendorSha256` value within `flake.nix`.
+
+If `flake.nix` changes or is modified you will also need to update the
+`flake.lock` file. You can do this by running: `nix flake update`. It is
+also recommended to then run `nix build` after this operation to ensure
+the build will be successful.
+
+## Testing ##
+
+You can execute tests for the `example` package by running the following
+command from within the `./skeleton-golang-package/src/example` directory:
+
+```console
+cd ./skeleton-golang-package/src/example
```

Comment:
It is a fairly standard practice to make all references relative to the project's root.  It's also one less place where a downstream repo has to worry about changing references from `skeleton-golang-package` to their repo name.

```suggestion
command from within the `src/example` directory:

```console
cd src/example
```